src/strategy/indicators/sar.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { SAR } from './sar'
+import { candles } from './spec.helper'
+
+describe('SAR', () => {
```

Comment:
This test suite, does only cover 65% of the SAR calculation.
Could you please add more test cases? I think we should aim for a higher test coverage than 60% (i personally try to reach 100% for my commits).
You will find the test coverage for you Pull-Request as attachment.
[sar.ts.html.zip](https://github.com/magic8bot/magic8bot/files/2260220/sar.ts.html.zip)

i would add an down trend calculation, the null check (not enough data) and  accel reset